pkgs/tools/misc/qmk/default.nix

```diff
@@ -27,13 +39,23 @@ buildPythonApplication rec {
     appdirs
     argcomplete
     colorama
-    dotty-dict
+    #dotty-dict # See https://pypi.org/project/qmk-dotty-dict/
+    qmk-dotty-dict
     hid
     hjson
     jsonschema
     milc
     pygments
+    pyrsistent
```

Comment:
https://github.com/NixOS/nixpkgs/pull/136976 updated qmk in Nixpkgs and it looks like pyrsistent is no longer needed: https://github.com/qmk/qmk_cli/commit/8e1de7491882cccaa7d7c66fbad0c0c0cd059474